pos_invoice_journal/__openerp__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Pos Invoice Journal',
+    'summary': """
```

Comment:
Here, too, I used https://github.com/acsone/bobtemplates.odoo/blob/master/bobtemplates/odoo/addon/%2Baddon.name%2B/__openerp__.py.bob#L8-L9

@sbidoul would you like to open an issue on https://github.com/acsone/bobtemplates.odoo ? By the way, thanks a lot for this tool, very helpful!